silverbackis/common-js-bundle/1.0/config/packages/common_js.yaml

```diff
@@ -0,0 +1,17 @@
+# Enable and configure the scripts you'd like
+common_js:
+    google_analytics:
+        enabled: false
+        debug: '%kernel.debug'
```

Comment:
I see that your bundle is not released yet. And by default `debug` is `false`, but there you set another default value. It's not late to do this functionality by default in bundle config. My suggestion is to do that in: https://github.com/silverbackis/common-js-bundle/blob/master/DependencyInjection/Configuration.php
See examples here:
https://github.com/8p/EightPointsGuzzleBundle/blob/master/src/DependencyInjection/Configuration.php#L52
https://github.com/schmittjoh/JMSSerializerBundle/blob/master/DependencyInjection/Configuration.php#L195

Reply:
As I switched off yesterday though that too - I'll set the default as suggested